krun/tests/test_linuxplatform.py

```diff
@@ -439,6 +448,8 @@ def test_check_dmesg_filter0001(self, platform, mock_manifest):
             platform.get_dmesg_whitelist(), old_lines, new_lines, mock_manifest)
 
     def test_aslr0001(self, platform, caplog):
+        caplog.set_level(logging.DEBUG)
```

Comment:
Is this intentional?

Reply:
Yeah. caplog used to capture all messages by default, but now it doesn't capture debug-level by default. Some tests check the debug level messages, so we have to turn them on.

Reply:
OK, we should tweak the commit message a bit?

Reply:
Sure. I'll include it at squash time.